src/overlays/actors/ovl_Bg_Astr_Bombwall/z_bg_astr_bombwall.h

```diff
@@ -6,14 +6,15 @@
 struct BgAstrBombwall;
 
 typedef void (*BgAstrBombwallActionFunc)(struct BgAstrBombwall*, GlobalContext*);
-
 typedef struct BgAstrBombwall {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x18];
-    /* 0x015C */ BgAstrBombwallActionFunc actionFunc;
-    /* 0x0160 */ char unk_160[0xDC];
-} BgAstrBombwall; // size = 0x23C
-
+   /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ BgAstrBombwallActionFunc actionFunc;
+    /* 0x160 */ ColliderTris unk160;                /* inferred */
+    /* 0x180 */ ColliderTrisElement unk180;         /* inferred */
+    /* 0x1DC */ char pad1DC[0x5C];
+    /* 0x238 */ s16 unk238;                         /* inferred */
+    /* 0x23A */ char pad23A[2];
```

Comment:
```suggestion
   /* 0x000 */ DynaPolyActor dyna;
    /* 0x15C */ BgAstrBombwallActionFunc actionFunc;
    /* 0x160 */ ColliderTris collider;
    /* 0x180 */ ColliderTrisElement colliderElements;
    /* 0x1DC */ UNK_TYPE1 pad1DC[0x5C];
    /* 0x238 */ s16 unk238;
    /* 0x23A */ UNK_TYPE1 pad23A[2];
```
Just as a suggestion these are the naming conventions we use for these types of variables. Also, I want to say `pad1DC` might be some of the colliderElements, but I haven't looked that closely.